exporter/cat1_view_helper.go

```diff
@@ -15,29 +15,20 @@ import (
 	"github.com/projectcypress/cdatools/models"
 )
 
-func escape(i interface{}) string {
-	switch str := i.(type) {
-	case string:
-		return escapeString(str)
-	case int64:
-		return escapeString(strconv.FormatInt(str, 10))
-	case int:
-		return escapeString(strconv.Itoa(str))
-	case *int64:
-		if str != nil {
-			return escapeString(strconv.FormatInt(*str, 10))
-		}
-	case *int:
-		if str != nil {
-			return escapeString(strconv.Itoa(*str))
-		}
-	}
-	return ""
+// int64PtrToString converts the value of an *int64 to a string.
+// NOTE: This function is probably not even needed, since I believe
+// text/template will access the value *int64 points to and not the
+// actual address.
```

Comment:
Do we get an error if the value is null? Or does it just end up being left blank? So in the case of _author.xml would we have to check if Time is nil before printing something?